packages/api/src/hooks/UseCennznet.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019-2020 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { web3Accounts, web3Enable } from '@polkadot/extension-dapp';
+import { InjectedAccountWithMeta, MetadataDef } from '@polkadot/extension-inject/types';
+import { getSpecTypes } from '@polkadot/types-known';
+import { defaults as addressDefaults } from '@polkadot/util-crypto/address/defaults';
+
+import { ApiOptions } from '../types';
+import { Api, Api as ApiPromise } from '../Api';
```

Comment:
will this only provide the promise version of API and not rx version?

Reply:
added option to return rx api as well

Reply:
consider using another function instead of bool
e.g.
```ts
UseCennznet
UseCennznetRx
```

Reply:
hmm yeah, probably more inline with our current API

Reply:
cool updated now